README.md

```diff
@@ -30,4 +30,9 @@ $factory = new EPoetryClientFactory('http://europa.eu/epoetry.wsdl', $adapter);
 $createRequests = new CreateRequests();
 $response = $factory->getClient()->createRequests($createRequests);
 ```
+## Authentication
 
+Access to ePoetry web services requires your application to be
+configured as a [cas proxy](https://webgate.ec.europa.eu/CITnet/confluence/display/IAM/Proxy+Tickets).
+
+When authenticated, a session needs to be provided having the attribute `cas_pgt`.
```

Comment:
We should extend this documentation with more details on how to instantiate the middleware and add it to the ePoetry client.

Reply:
Done.